src/Uno.Toolkit.UITest/Controls/SafeArea/Given_SafeArea.cs

```diff
@@ -1,199 +1,201 @@
-
-			(var mainElement, var sampleName) = safeAreaType.ToLower() switch
-			{
-				"control" => ("MySafeArea", "SafeArea_Control_NestedPage"),
-				"attached" => ("MyBorder", "SafeAreaSamplePage_NestedPage"),
-				_ => throw new InvalidOperationException($"{nameof(safeAreaType)} does not support a value of: {safeAreaType}")
-			};
-
-			NavigateToNestedSample(sampleName);
-			App.WaitForElementWithMessage(mainElement);
-
-			var isMargin = string.Equals(mode, "Margin", StringComparison.OrdinalIgnoreCase);
-
-			App.FastTap($"{mode}Mode");
-			App.FastTap($"SetAllTwenty");
-
-
-			foreach (List<string> maskCombo in Subsets(_availableMasks))
-			{
-				ClearMasks();
-
-				foreach (var mask in maskCombo)
-				{
-					App.FastTap($"{mask}{CheckboxSuffix}");
-				}
-
-				var testRect = App.GetPhysicalRect(isMargin ? "WrappingGrid" : mainElement);
-				using var screenshot = TakeScreenshot($"Inset{string.Join("_", maskCombo)}_Override_{InsetThickness}");
-
-				if (HasMask(Left))
-					ImageAssert.HasPixels(
-						screenshot,
-						ExpectedPixels
-							.At(testRect.X + HalfBorderThickness, testRect.Height / 2)
-							.Named($"Inset{string.Join("_", maskCombo)}_Override_{InsetThickness}_Left_Inset")
-							.Pixel(GetExpectedColor(Left))
-							.WithColorTolerance(5)
-					);
-				if (HasMask(Top))
-					ImageAssert.HasPixels(
-						screenshot,
-						ExpectedPixels
-							.At(testRect.Width / 2, testRect.Y + HalfBorderThickness)
-							.Named($"Inset{string.Join("_", maskCombo)}_Override_{InsetThickness}_Top_Inset")
-							.Pixel(GetExpectedColor(Top))
-							.WithColorTolerance(5)
-					);
-				if (HasMask(Right))
-					ImageAssert.HasPixels(
-						screenshot,
-						ExpectedPixels
-							.At(testRect.Width - HalfBorderThickness, testRect.Height / 2)
-							.Named($"Inset{string.Join("_", maskCombo)}_Override_{InsetThickness}_Right_Inset")
-							.Pixel(GetExpectedColor(Right))
-							.WithColorTolerance(5)
-					);
-				if (HasMask(Bottom))
-					ImageAssert.HasPixels(
-						screenshot,
-						ExpectedPixels
-							.At(testRect.Width / 2, testRect.Bottom - HalfBorderThickness)
-							.Named($"Inset{string.Join("_", maskCombo)}_Override_{InsetThickness}_Bottom_Inset")
-							.Pixel(GetExpectedColor(Bottom))
-							.WithColorTolerance(5)
-					);
-
-				bool HasMask(string mask) =>
-					maskCombo.Contains("All", StringComparer.OrdinalIgnoreCase) ||
-					(!mask.Equals("SoftInput", StringComparison.OrdinalIgnoreCase) && maskCombo.Contains("VisibleBounds", StringComparer.OrdinalIgnoreCase)) ||
-					maskCombo.Contains(mask, StringComparer.OrdinalIgnoreCase);
-
-				Color GetExpectedColor(string mask) => HasMask(mask) ? GetInsetColor() : Color.Red;
-
-				Color GetInsetColor()
-				{
-					return string.Equals(mode, "Padding", StringComparison.OrdinalIgnoreCase) ? Color.Yellow : Color.Green;
-				}
-			}
-		}
-
-		[Test]
-		[ActivePlatforms(Platform.iOS)]
+﻿using NUnit.Framework;

```

Comment:
care to fix the diff here?

Reply:
Uh, wait what the hell is happening here

Reply:
ftfy